lib/travis/logs/drain_consumer.rb

```diff
@@ -128,7 +112,7 @@ def dead?
           end
 
           begin
-            safe_ack(delivery_tag)
+            ack(delivery_tag)
           rescue StandardError => e
```

Comment:
this is where the exception will bubble up, which will push things back into the batch buffer, so that they get re-tried the next time around.